src/test/java/com/sap/sgs/phosphor/fosstars/model/value/DateValueTest.java

```diff
@@ -62,4 +62,4 @@ public void serialization() throws IOException {
     assertEquals(dateValue.hashCode(), clone.hashCode());
   }
 
-}
\ No newline at end of file
+}
```

Comment:
Is there any change in this file?

Reply:
There used to be changes but not anymore. I'll revert this.